db/migrate/20170125015659_create_districts.rb

```diff
@@ -1,4 +1,8 @@
 class CreateDistricts < ActiveRecord::Migration
+
+  enable_extension "postgis"
+  enable_extension "pgrouting"
```

Comment:
I think these need to be inside `def change` see http://stackoverflow.com/questions/16611226/how-to-install-postgres-extensions-at-database-creation